cortx-motr.spec.in

```diff
@@ -2,7 +2,7 @@
 %bcond_with ut
 %bcond_with debuginfo_sources
 %bcond_with cassandra
-%bcond_with isal
+%bcond_without isal
```

Comment:
But this condition is not used in this spec file.
The "--with isal" is still used here.
Please update this.

Reply:
`%bcond_without isal` will enable isal by default. Hence the check `%if %{with isal}` will become true and it will install Intel ISA library. 
Ref: https://rpm.org/user_doc/conditional_builds.html